### PR TITLE
Add contentType filter on site pages query

### PR DESCRIPTION
**Pull request by sks444**
Closes https://github.com/GrappleGQL/wagtail-grapple/issues/170

### CODE REVIEW

**Comment by zerolab** [comment]
Thank you for this. Could you add a couple of tests too?

**Comment by sks444** [comment]
@zerolab added tests for this.

File: grapple/types/sites.py [hunk 1]
```diff
@@ -21,9 +29,21 @@ class SiteObjectType(DjangoObjectType):
...
-            WagtailPage.objects.in_site(self).live().public().specific(), info, **kwargs
-        )
+        pages = WagtailPage.objects.in_site(self).live().public().specific()
+
+        content_type = kwargs.pop("content_type", None)
+        if content_type:
+            app_label, model = content_type.strip().lower().split(".")
+            try:
+                ctype = ContentType.objects.get(app_label=app_label, model=model)
+            except:  # noqa
```
Comment (zerolab) [inline grapple/types/sites.py:39]:
```suggestion
            except ContentType.DoesNotExist: 
```

**Comment by zerolab** [comment]
LInted manually and merged. Thank you

**Comment by sks444** [comment]
Thanks for the merge @zerolab!

**Pull request closed without merging**